src/Icon/index.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+
+import styles from './Icon.scss';
+
+function Icon(props) {
+  return (
+    <div>
+      <span
+        id={props.id}
+        className={classNames(props.className)}
+        aria-describedby={props.describedBy}
+        aria-hidden={props.hidden}
+      />
+      { props.srText &&
+        <span className={classNames(styles['sr-only'])}>
+            {props.srText}
+        </span>
+      }
+    </div>
+  );
+}
+
+Icon.propTypes = {
+  id: PropTypes.string.isRequired,
+  className: PropTypes.arrayOf(PropTypes.string).isRequired,
+  describedBy: PropTypes.string,
```

Comment:
@cptvitamin does it ever make sense for an Icon to have an aria-describedBy attribute? I was trying to create a story for this to demo it, but was having a hard time thinking of when this would be used.

Reply:
@fysheets nothing [explicitly prevents](https://www.w3.org/TR/html-aria/#allowed-aria-roles-states-and-properties) aria-describedby from being used on any element.  That said, I personally can't think of a valid use case for using it on an icon...  Icons are usually used to convey something pretty concise, like STOP, WARNING, ERROR, EMAIL, UNREAD etc.  If the icon requires more nuance to describe it, i would think that nuance would be in accessible text that is adjacent to the icon.